src/styles.css

```diff
@@ -0,0 +1,201 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
```

Comment:
Ich würde keine Google Fonts so einbinden, da es nicht Datenschutz-konform ist. Sprich, wenn man das auf einer öffentliche Website so macht, kann man von einem Anwalt abgemahnt werden deswegen.

Siehe dazu auch: [https://github.com/neuefische/hh-web-23-3/blob/main/sessions/css-basics/css-basics.md](https://github.com/neuefische/hh-web-23-3/blob/main/sessions/css-basics/css-basics.md) und da in der Rubrik: "Legal Issues using Google Fonts with @import and <link>"

Reply:
Danke, ich schaue es mir an 👌🏼